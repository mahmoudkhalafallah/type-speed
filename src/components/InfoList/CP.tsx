import React from 'react'
const CP: React.FC<{ cp: number }> = ({ cp }) =>
    <>
        Completion Percentage:
        <span>{cp}%</span>
    </>
export default CP