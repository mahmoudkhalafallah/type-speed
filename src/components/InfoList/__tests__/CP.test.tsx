import React from 'react'
import CP from '../CP'
import { render } from '@testing-library/react'

test('CP rendered correctly', () => {
    const { baseElement, getByText } = render(
        <CP
            cp={90}
        />,
    )
    expect(getByText('90%')).toBeInTheDocument()
    expect(baseElement).toMatchSnapshot()
})