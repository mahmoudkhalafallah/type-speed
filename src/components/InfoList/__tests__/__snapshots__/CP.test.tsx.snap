// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CP rendered correctly 1`] = `
<body>
  <div>
    Completion Percentage:
    <span>
      90
      %
    </span>
  </div>
</body>
`;
